book/installation.rst

```diff
@@ -269,6 +269,14 @@ If there are any issues, correct them now before moving on.
 
         setfacl isn't available on NFS mount points. However, setting cache
         and logs over NFS is strongly not recommended for performance.
+        
+    .. note::
+
+        When not using PHP's built-in web server, make sure that the web server
+        process runs with your user account or to configure advanced rules (like ACLs).
+        
+        http://symfony.com/doc/current/cookbook/configuration/web_server_configuration.html.
+        
```

Comment:
Original though was to point out in the note that setfacl should also check for the user which is being used not only by webserver, but also possibly by php-fpm. The latter uses www-user instead of www-data by default which causes write permission issues.

I think it's safe to reject it & close this pull request.